lib/command_utils.js

```diff
@@ -106,7 +108,7 @@ const extractCommand = command => {
       extractedCommand = result;
     }
   } else {
-    extractedCommand = command.query || command;
+    extractedCommand = command.query ? deepCopy(command.query) : deepCopy(command);
```

Comment:
```suggestion
    extractedCommand = deepCopy(command.query || command);
```
I'd keep the more concise || syntax from the original code.